roles/dropwizard-api/files/supervisord-run.sh

```diff
@@ -12,11 +12,28 @@ name=$(basename "$(pwd)")
 node="$(whoami)@$(hostname):$name"
 archive=$(ls "$name"-*-all.jar | sort | tail -1)
 config=/apis/config/$name.yaml
+env=/apis/apis/$name/environment.env
 
 if [ ! -e "$config" ]; then
     config=/apis/apis/$name/configuration.yaml
 fi
 
+# Read environment file
+# Only allow variables that start with a capital letter
+# to prevent them from overwriting our own variables.
+# Don't allow overwriting the PATH variable.
+if [ -r "$env" ]; then
+    while IFS='\n' read -r line; do
+        case "$line" in
+        PATH=*)
+            ;;
+        [A-Z]*=*)
+            export "$line"
+            ;;
+        esac
```

Comment:
Maybe. Although from experience, dropwizard-template-config will complain loudly if you try to reference a variable which isn't set, so i don't know if an extra warning here would help much. The problems i actually ran into when testing this were typos in variable names and the file being in the wrong location.